subcommand/acl-init/command.go

```diff
@@ -127,7 +128,7 @@ func (c *Command) Run(args []string) int {
 }
 
 func (c *Command) getSecret(secretName string) (string, error) {
-	secret, err := c.k8sClient.CoreV1().Secrets(c.flagNamespace).Get(secretName, metav1.GetOptions{})
+	secret, err := c.k8sClient.CoreV1().Secrets(c.flagNamespace).Get(context.Background(), secretName, metav1.GetOptions{})
```

Comment:
```suggestion
	secret, err := c.k8sClient.CoreV1().Secrets(c.flagNamespace).Get(context.TODO(), secretName, metav1.GetOptions{})
```

Reply:
I did see those and skipped them intentionally. Given these subcommands are `main` files that we invoke Run(args) on, would we want to pass context into them in the future? I assumed they would behave provide the context themselves

Reply:
I figured we'd start a context.Background() at the top and then feed it through to these helpers.